### PR TITLE
QA requests

### PR DESCRIPTION
**Pull request by buddy-web3**

### CODE REVIEW

File: app/newRoot.css [hunk 1]
```diff
@@ -529,3 +529,10 @@ ol {
     flex-direction: column;
   }
 } /* end mobile */
+p,
+button {
+  font-family: Poppins;
+}
+.margin-top-40 {
```
Comment (melissasamworth) [inline app/newRoot.css:536]: we have a padding top class. Do you need margin specifically?

File: app/newRoot.css [hunk 2]
```diff
@@ -529,3 +529,10 @@ ol {
     flex-direction: column;
   }
 } /* end mobile */
+p,
+button {
+  font-family: Poppins;
```
Comment (melissasamworth) [inline app/newRoot.css:534]: this is specified in body, why does it need to be specified again?

**Review by melissasamworth — CHANGES_REQUESTED** [review]

File: app/newRoot.css (hunk 1 above)
Comment (buddy-web3) [reply app/newRoot.css:536]: yes

File: app/newRoot.css (hunk 2 above)
Comment (buddy-web3) [reply app/newRoot.css:534]: yes. that's why buttons still are Arial

**Review by melissasamworth — APPROVED** [review]

**Pull request merged**